### DISCUSSION TITLE
Allow Full Range Logs Volume to be disabled via data source settings

### DISCUSSION THREAD

**Discussion by vjsamuel** [grafana/grafana#70215] [Explore]
Full Range Logs Volume can be a very expensive query. In an environment where there are 100s of end users using a Grafana installation, it is very hard to educate all of them to collapse that panel on Explore to prevent overloading the backend. Is there a possibility to provide a disable feature button on the data source setting of Loki? Such is already being done for things like fetching list of metric names on the Prometheus data source today.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82647. Feel free to continue the discussion around this there. Thank you!
